os/kernel/binary_manager/binary_manager_recovery.c

```diff
@@ -92,7 +92,7 @@ static void binary_manager_reset_board(void)
 	(void)irqsave();
 	sched_lock();
 	for (;;) {
-		lldbg("\nASSERT!! Push the reset button!\n");
+		printf("\nASSERT!! Push the reset button!\n");
```

Comment:
This is the assert code so that lldbg is correct. Let's revert this.

Reply:
How about adding comment about the reason why here uses lldbg instead of printf?

Reply:
Reverted and added the reason why changed to printf in the commit msg.
Thanks.